Text/AbbyyOCR/AbbyyOCR.cs

```diff
@@ -0,0 +1,257 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using AzureCognitiveSearch.PowerSkills.Common;
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+using Abbyy.CloudSdk.V2.Client.Models;
+using Microsoft.Extensions.DependencyInjection;
+using System.Net.Http;
+using Abbyy.CloudSdk.V2.Client;
+using Abbyy.CloudSdk.V2.Client.Models.RequestParams;
+using Abbyy.CloudSdk.V2.Client.Models.Enums;
+using System.Net;
+using Polly.Extensions.Http;
+using Polly;
+using System.Collections.Concurrent;
+
+namespace AbbyyOCR
+{
+    public static class AbbyyOCR
+    {
+        private const string ApplicationId = @"[Enter ABBYY Application ID]";
+        private const string Password = @"[Enter ABBYY OCR Password]";
+        private const string ServiceUrl = "[Enter ABBYY Service URL such as https://cloud-westus.ocrsdk.com]";
+
```

Comment:
For consistency with other skills in the repo, please make those constants be names of environment variables that contain the actual values. See https://github.com/Azure-Samples/azure-search-power-skills/blob/ccebef7e72055ef1ba44fd058e21d6deb6a01474/Text/BingEntitySearch/BingEntitySearch.cs#L20 for an example.